Description: Build hiro without SourceEdit to avoid gtksourceview dependence
 gtksourceview2 is deprecated and will be removed from Debian.
Author: Tobias Hansen <thansen@debian.org>
Bug-Debian: http://bugs.debian.org/885687

--- a/hiro/GNUmakefile
+++ b/hiro/GNUmakefile
@@ -36,8 +36,8 @@
   endif
 
   ifeq ($(hiro),gtk)
-    hiroflags = $(cppflags) $(flags) -DHIRO_GTK $(shell pkg-config --cflags gtk+-2.0 gtksourceview-2.0)
-    hirolink = -lX11 $(shell pkg-config --libs gtk+-2.0 gtksourceview-2.0)
+    hiroflags = $(cppflags) $(flags) -DHIRO_GTK $(shell pkg-config --cflags gtk+-2.0)
+    hirolink = -lX11 $(shell pkg-config --libs gtk+-2.0)
   endif
 
   ifeq ($(hiro),qt)
--- a/hiro/components.hpp
+++ b/hiro/components.hpp
@@ -66,7 +66,6 @@
 #define Hiro_ProgressBar
 #define Hiro_RadioButton
 #define Hiro_RadioLabel
-#define Hiro_SourceEdit
 #define Hiro_TabFrame
 #define Hiro_TableView
 #define Hiro_TextEdit
